imports/plugins/core/dashboard/client/templates/shop/settings/settings.js

```diff
@@ -76,12 +76,16 @@ Template.shopSettings.helpers({
     return "";
   },
   brandImageSelectProps() {
+    const shopId = Reaction.getShopId();
+
     const media = Media.find({
+      "metadata.ownerId": Meteor.userId(),
```

Comment:
would this limit the query to only those that the current user uploaded?

Reply:
@pmn4, that's what it does.

Reply:
ok, got it.
this is a super minor scenario, but do you think this could be an issue in shops with multiple admins? i.e., what if I mistakenly change the brand image and want to get it back to the image that you uploaded? I would have to ask you to change it back, right?

Reply:
Yes, that would be case. I would changing that soon, need to remove that line.